### DISCUSSION TITLE
Pulled the v5.0.3 tag. Every built package and PKGBUILD have 4.8.2

### DISCUSSION THREAD

**Discussion by DDzwiedziu** [wwmm/easyeffects#960] [Q&A]
Is it a bug or something I'm to buildpackage-noob to understand?

![Screenshot_20210507_134350](https://user-images.githubusercontent.com/3499420/117445040-850b3400-af3a-11eb-8039-559a2d0e6811.png)

**Comment by wwmm** [comment]
>Is it a bug or something I'm to buildpackage-noob to understand?

In a way it is. What is happening is that the scripts for Debian packages in the master branch have not been updated for the PipeWire release yet. They are still the same as the ones for Pulseaudio. As I do not use a Debian based distribution there isn't much I can do about this. These scripts are kept here just for Debian package maintainers convenience.

If all you want is the latest release for Pulseaudio use this branch https://github.com/wwmm/pulseeffects/tree/pulseaudio-legacy. But if you really want to use our PipeWire release I am not sure about the best way to handle this in Debian based distributions.

**Reply by DDzwiedziu** [reply]
I've bumped the version manually (but in `debian/changelog`, not `PKGBUILD`). It compiles, installs, works. So I'm not sure what to make of it.
